partials/tabs/about.html

```diff
@@ -129,3 +129,4 @@ <h2 class="fine center">about</h2>
   $('ul.tabs').tabs();
 });
 </script>
+<link rel="stylesheet" href="/resources/css/people.css">
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmph_inr9rr/partials/tabs/about.html
+++ b/tmp/tmph_inr9rr/partials/tabs/about.html
@@ -129,4 +129,4 @@
   $('ul.tabs').tabs();
 });
 </script>
-<link rel="stylesheet" href="/resources/css/people.css">+<link rel="stylesheet" href="/resources/css/people.css">
```